scripts/cnv_wdl/germline/joint_call_exome_cnvs.wdl

```diff
@@ -151,7 +151,7 @@ workflow JointCallExomeCNVs {
     }
     Array[String] subset_arr = select_all(subset)
     Array[String] subset_index_arr = select_all(subset_indexes)
-    Array[String] failed_samples = select_all(failed)
+    Array[String] failed_samples_arr = select_all(failed)
```

Comment:
Changed the name of this (internal) variable as it is also defined as an output and was failing validation for that reason